generate-stackbrew-pr.sh

```diff
@@ -90,9 +90,10 @@ function pr_payload() {
 
   printf "{
     \"title\": \"Node: %s\",
-    \"body\" : \"%s.\n cc %s\",
+    \"body\" : \"%s.<br />cc %s\",
```

Comment:
Yeah l! The PR that was created on the official images uses this code and it works perfectly. For some reason it doesn't accept new lines but it does accept html (it probably accepts Markdown as well)

Reply:
Maybe use `\n` instead of `\\n` will work.

Ref:
https://github.com/cdnjs/buildScript/blob/master/build.sh#L26-L28
https://github.com/cdnjs/cdnjs/issues/12973